src/components/MenuItem.js

```diff
@@ -8,8 +8,12 @@ import styles, {getButtonBackgroundColorStyle, getIconFillColor} from '../styles
 import Icon from './Icon';
 import {ArrowRight} from './Icon/Expensicons';
 import getButtonState from '../libs/getButtonState';
+import Badge from './Badge';
 
 const propTypes = {
+    // Text to be shown as badge near the right end.
```

Comment:
This comment should use block comments per [style guide](https://github.com/Expensify/App/blob/main/STYLE.md#method-naming-and-code-documentation)